config/locales/custom/verification.yml

```diff
@@ -5,12 +5,14 @@ en:
       new:
         accept_terms_adult: I hereby declare on my own responsability that I am at least 18 years old.
         accept_terms_resident: I hereby declare on my own responsability that I reside permanently or temporarily in %{residence}.
+        city_name: Dumbrăvița
         accept_terms_terms: I accept %{terms_url} and %{privacy_url}.
         terms: the Terms and Conditions
         privacy: the Privacy Policy
         help_needed: |
           If you run into any issues with the verification process, contact us: %{email}
-        explanation: We are using your CNP only to validate that you are a citizen of Brasov, as stated in %{rules_url}.
+        support_email: contact@primaria-dumbravita.ro
+        explanation: We are using your CNP only to validate that you are a citizen of %{residence}, as stated in %{rules_url}.
```

Comment:
Suggest "We are using your ID number only to validate that you are a citizen of %{residence}, as stated in %{rules_url}."
since other countries might not have the same documents used for identification

Reply:
Hi @decembrya 

Ok, we'll update it right now. 

So, the Romanian text doesn't need to be changed? Just change the English text?  
```
explanation: Folosim CNP-ul dumneavoastră doar pentru a valida faptul că sunteți rezident în comuna %{residence}, conform %{rules_url}.
```

Reply:
Yeah, Romanian looks good 👍